test/language/statements/let/let-let-declaration-split-across-two-lines.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2015 Mozilla Corporation. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+author: Jeff Walden
+es6id: 13.3
+flags: noStrict
+description: >
+    let: |let let| split across two lines is not subject to automatic semicolon insertion.
```

Comment:
If this is the case being tested, doesn't the following suffice: 

```js
let
let = 'parsed';
```

And that should throw?